### ISSUE TITLE
[OSX 10.14.6] Cannot launch the game

### ISSUE THREAD

**Issue by midgardormr** [cataclysmbnteam/Cataclysm-BN#524]
# Describe the bug

Hello! Wanted to give BN a try instead of DDA, but just cannot launch the game at all...
No idea what I can do to play it. Tried few releases, both tiled and non-tiled versions.

# Steps To Reproduce

1. Download latest OSX release on 'Releases' page.
2. Move 'Cataclysm.app' into Applications folder.
3. Disable FileVault and try to open app.
4. Nothing is happening% no crash logs, no errors, just nothing.

# Expected behavior
I can launch CDDA, I have been playing it for month. Nothing strange was happening with CDDA app. Why I cannot launch BN app?

# Screenshots

# Versions and configuration

# Additional context

**Comment by olanti-p** [comment]
After some discussion on Discord, it was determined that the game ignores dynamic libraries bundled with the `.dmg` file, and instead tries to load them from `/usr/local/opt`. For example, CURSES version fails with:
```
dyld: Library not loaded: /usr/local/opt/ncurses/lib/libncursesw.6.dylib
  Referenced from: /Applications/Cataclysm 2.app/Contents/Resources/./cataclysm
  Reason: image not found
```
That directory is typically used by Homebrew - and, indeed, after installing Homebrew and running
```
brew install gettext sdl2 sdl2_image sdl2_ttf sdl2_mixer
```
TILES version launched without an issue.

I'm really not familiar with OSX or Homebrew. The libraries seem to be where they should be, at least on TILES
![image](https://user-images.githubusercontent.com/60584843/120119367-0e133500-c1a0-11eb-944f-c48ae410e005.png)
CURSES version only has `libintl`, but `ncursesw` is TTBOMK comes preinstalled on Macs.
![image](https://user-images.githubusercontent.com/60584843/120119620-6139b780-c1a1-11eb-9970-60877d124553.png)

**Comment by olanti-p** [comment]
Possible fix: https://github.com/CleverRaven/Cataclysm-DDA/pull/50234
May be relevant: https://github.com/CleverRaven/Cataclysm-DDA/pull/49595

**Comment by Coolthulhu** [comment]
Does this still reproduce?

**Comment by Osorkonius** [comment]
Can confirm that it still reproduces

**Comment by Firestorm01X2** [comment]
I have one more report about similar problem. Problem is definitely still relevant.

**Comment by Coolthulhu** [comment]
Assuming it has been fixed, since it seems to be the same as #1297
